apps/src/templates/sectionsRefresh/SectionsSetUpContainer.jsx

```diff
@@ -425,6 +425,12 @@ const styles = {
     verticalAlign: 'text-bottom',
   },
   tooltipIcon: {color: color.neutral_dark60, fontSize: '16px'},
+  toolTipBox: {
```

Comment:
Are these styles copied from the other component where we have tooltips on this page? Would it be possible to factor it out and refer to the same thing from both places so we reduce the risk of the styles diverging in the future?

Reply:
I was initially hoping we could utilize the `InfoHelpTip` component in this space, but when I tried doing that, it lead to some weird css issues which led me to this solution. 

I did just re-push some code that instead changes the `InfoHelpTip` component so that it works in both spots (both in the Co-teacher area and the Advanced Settings area).   It required more custom-css for the `InfoHelpTip` but as a result it works more consistently in more areas of the form (and perhaps in other areas of the website too).